domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2379,26 +2379,42 @@ private FacetManager buildFacetManager(CQLWrapper wrapper, QueryHelper queryHelp
 
   /**
    * Get the jsonb by id.
+   * @param conn  if provided, the connection on which to execute the query on.
```

Comment:
The description should explain all three cases:
* if provided, the connection to use
* if null, a new connection is created and automatically released afterwards
* if failed, the replyHandler is failed